test/e2e/storage/vsphere/vsphere_volume_fstype.go

```diff
@@ -75,8 +73,7 @@ var _ = utils.SIGDescribe("Volume FStype [Feature:vsphere]", func() {
 		Bootstrap(f)
 		client = f.ClientSet
 		namespace = f.Namespace.Name
-		nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
-		Expect(len(nodeList.Items)).NotTo(BeZero(), "Unable to find ready and schedulable Node")
+		Expect(framework.GetReadySchedulableNodesOrDie(f.ClientSet).Items).NotTo(BeEmpty(), "Unable to find ready and schedulable Node")
```

Comment:
DItto on getting ready nodes vs using that list from TestContext.

Reply:
replaced with `Expect(GetReadySchedulableNodeInfos).NotTo(BeEmpty())`